src/docs/index.md

```diff
@@ -261,20 +305,31 @@ Firepad also provides a headless mode for interacting with documents without a G
 and in your code:
 
 {% highlight javascript %}
-var Firepad = require('firepad')
-var headless = new Firepad.Headless('<FIREBASE_URL>')
+var Firepad = require('firepad');
+var headless = new Firepad.Headless('https://<DATABASE_NAME>.firebaseio.com');
```

Comment:
It seems like this probably doesn't work anymore...  unless headless is still using the 2.x client.